feg/gateway/services/session_proxy/credit_control/gx/gx_client.go

```diff
@@ -430,15 +430,15 @@ func (gxClient *GxClient) getEventTriggerAVP(eventTrigger EventTrigger) *diam.AV
 	return diam.NewAVP(avp.EventTrigger, avp.Mbit|avp.Vbit, diameter.Vendor3GPP, datatype.Enumerated(eventTrigger))
 }
 
-// getAPNfromConfig returns a new apn value to overwrite the one in the request based on list of regex definied in Gx config.
+// getAPNFromConfig returns a new apn value to overwrite the one in the request based on list of regex definied in Gx config.
```

Comment:
Golang convention (in Magma codebase and in stdlib) is to wrap comments to 79 chars